package.json

```diff
@@ -35,6 +38,7 @@
     "select2": "~3.5.2-browserify"
   },
   "scripts": {
-    "lint": "eslint -c ./.eslintrc webpack/assets/**/*.js  || exit 0"
+    "lint": "./node_modules/.bin/eslint -c .eslintrc webpack/assets/**/*.js",
+    "jest": "jest"
```

Comment:
would suggest naming the script "test" instead of jest. Then you can run "npm test

jest can also just be run directly from the command line